taworks/taform/templates/taform/application.html

```diff
@@ -68,7 +68,12 @@ <h3>Basic Information</h3>
         <td>{{s_form.ta_expectations}}&nbsp;Yes</td>
         <td>{{s_form.ta_expectations.help_text}}</td> 
      </tr>
+     <tr>
+        <td>Preference of Position Type (select all that apply):</td>
+        <td>{{s_form.full_ta}}&nbsp;Full TA&emsp;{{s_form.half_ta}}&nbsp;1/2 TA</td>
```

Comment:
sort of picky but do we want to have it consistent like "full TA" and "half TA"?

Reply:
it's very asian to keep it full ta, 1/2 ta